src/config/layers.json

```diff
@@ -329,7 +329,8 @@
   "snow_groundstations": {
     "title": "Temperature (example data for 1-11 January 2020)",
     "type": "groundstation",
-    "data": "groundstationSnow",
+    "data": "http://119.40.103.16/temp/Temperature?beginDateTime=2020-03-01&endDateTime=2020-03-21",
```

Comment:
Should `data` contains `beginDateTime` and `endDateTime` params? Or should them be set is the app with data selector

Reply:
We'll keep it like this for now, if they want to improve it later, they can